src/plugins/recordTypes/objectexit/optionLists.js

```diff
@@ -3,28 +3,13 @@ import { defineMessages } from 'react-intl';
 export default {
   exitReasons: {
     values: [
-      'deaccession',
-      'disposal',
-      'returnofloan',
```

Comment:
You actually need to keep these values. The messages can be deleted because they'll be inherited, but the values won't. The values need to be repeated, because if they were merged, you wouldn't be able to control the ordering.

Reply:
Ah I see, I should have tested again before pushing but moved on to some other things.